NewForce-at-Mountwest/reactive-nutshell-twofoot
# Created News Article Feature

Created News feature. Added the ability to create, edit, and delete articles.

To Test:
1) Clear session and local storage
2) Create a user
3) you should see a page that says "News" with a 'New Article' button.
4) click the New Article button. You should see a form to make a new article.
5) Fill out the form.
6) Click Save Article. You should be back on the main news page with the article you just created.
7) Click the edit button on the article you just made.
8) Change some things and hit save article. You should see the changes on the home page.
9) Click Delete on the article. It should disappear.
10) Create several new articles, they should sort with the newest one on top.
